config/config.yaml

```diff
@@ -622,7 +622,7 @@ defaults:
   clustersService:
     image:
       registry: quay.io
-      repository: app-sre/uhc-clusters-service
+      repository: app-sre/aro-hcp-clusters-service
```

Comment:
This makes https://github.com/Azure/ARO-HCP/blob/49782426c3abd3bd498e94ced2c605eaafed9cba/config/config.yaml#L815 no longer needed